contracts/src/Creator.sol

```diff
@@ -13,17 +42,141 @@ contract Creator {
     // Storage
     // -----------------------------------------
 
+    /// @dev Streamers (that are a)
+    mapping (address => uint) public streamers;
+
     // -----------------------------------------
     // Constructor
     // -----------------------------------------
 
     // function postUpgrade(uint256 id) public proxied {}
 
-    constructor() {
-        
+    constructor() public ERC1155PresetMinterPauser() {
+        _mint(msg.sender, "{'name': 'test', 'description': 'testdesc', 'content': 'testurl'}", 1, ""); //1 means NFT.
+        ISuperfluid host,
+        IConstantFlowAgreementV1 cfa,
+        ISuperToken acceptedToken) {
+        assert(address(host) != address(0));
+        assert(address(cfa) != address(0));
+        assert(address(acceptedToken) != address(0));
+
+        _host = host;
+        _cfa = cfa;
+        _acceptedToken = acceptedToken;
```

Comment:
This is the stablecoin the creator is accepting as payment, right? Should we just hard code USDC for the sake of the hackathon?